onnxruntime/core/framework/allocation_planner.h

```diff
@@ -28,9 +28,14 @@ class ExecutionProviders;
 struct KernelCreateInfo;
 class KernelRegistryManager;
 class OrtValueNameIdxMap;
+class IStreamCommandHandleRegistry;
 
 using KernelCreateInfoMap = std::unordered_map<onnxruntime::NodeIndex, gsl::not_null<const KernelCreateInfo*>>;
 using SubgraphsKernelCreateInfoMaps = std::unordered_map<std::string, KernelCreateInfoMap>;
+// Specify how many logic streams for each provider type
+using ProviderStreamMap = InlinedHashMap<std::string, int>;
+// Each set contains ops which should be grouped in an independent logic stream
+using OpStreamMap = std::vector<std::vector<std::string>>;
```

Comment:
Not super clear what the contents of this vector<vector> are. Can you add some more documentation?

Reply:
this actually out of date and can be removed. will fix it.